core/metacling/src/TClingMethodInfo.h

```diff
@@ -53,35 +53,99 @@ namespace ROOT {
 class TClingClassInfo;
 class TClingTypeInfo;
 
+/// Iteration over Special Functions
+
+class TClingCXXRecMethIter final: public TClingMemberIter {
+
+class SpecFuncIter {
+   using Vec_t = llvm::SmallVector<clang::FunctionDecl *,4>;
+   Vec_t fDefDataSpecFuns; // Special functions materialized from DefinitionData.
+   size_t fIDefDataSpecFuns = 0; // Current element in fDefDataSpecFuns.
+
+public:
+   SpecFuncIter() = default;
+   SpecFuncIter(cling::Interpreter *interp, clang::DeclContext *DC,
+                llvm::SmallVectorImpl<clang::Decl*> &&specFuncs);
+
+   bool Valid() const { return fIDefDataSpecFuns < fDefDataSpecFuns.size(); }
```

Comment:
```suggestion
   bool IsValid() const { return fIDefDataSpecFuns < fDefDataSpecFuns.size(); }
```
everywhere else we use IsValid.